input/pagecontent/comparison.md

```diff
@@ -202,6 +202,44 @@ AU Core compliant resources are compliant with IPA requirements for Missing Data
 ##### Suppressed Data
```

Comment:
The primary table under the heading Profile comparison has not been updated and needs to be.

Reply:
Done

Reply:
Still needs to be updated, some IPS profiles need the yellow icon not the green based on the additional requirements table including patient & practitioner.

Reply:
Updated.